### PR TITLE
DRS Manifest and Cohort builder on Pilot data (syn50996868)

### PR DESCRIPTION
**Pull request by itismeghasyam**

### CODE REVIEW

**Review by pranavanba — APPROVED** [review]
Looks good so far

One thing to keep in mind- when developing for use with production data, reconsider using `dplyr::rowwise()` since it is less efficient than vectorized operations, e.g., `dplyr::rowwise() %>% dplyr::mutate()` will significantly impact performance on large datasets vs using just `dplyr::mutate()`.

File: cohort_builder.R [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+  `colnames<-`(c('parquet_pilot_path')) %>% 
+  dplyr::filter(grepl('SYNAPSE_METADATA', parquet_pilot_path))
+
+for(path_to_remove in pilot_files_to_remove$parquet_pilot_path){
+  file_to_remove <- paste0('./pilot_data/', path_to_remove)
+  print(file_to_remove)
+  unlink(file_to_remove)
+}
+
+rm(pilot_files_to_remove)
```
Comment (pranavanba) [inline cohort_builder.R:37]: Set `manifest="suppress"` in `syncFromSynapse()` to suppress creating manifest file automatically. See [synapseutils.sync.syncFromSynapse](https://python-docs.synapse.org/reference/synapse_utils/?h=syncfrom#synapseutils.sync.syncFromSynapse) for more details.

Comment (pranavanba) [inline cohort_builder.R:37]: Set `manifest="suppress"` in `syncFromSynapse()` to suppress creating manifest file automatically. See [synapseutils.sync.syncFromSynapse](https://python-docs.synapse.org/reference/synapse_utils/?h=syncfrom#synapseutils.sync.syncFromSynapse) for more details.

Comment (pranavanba) [inline cohort_builder.R:37]: Set `manifest="suppress"` in `syncFromSynapse()` to suppress creating manifest file automatically. See [synapseutils.sync.syncFromSynapse](https://python-docs.synapse.org/reference/synapse_utils/?h=syncfrom#synapseutils.sync.syncFromSynapse) for more details.

File: cohort_builder.R [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+# Need to partition at participant Identifier level, does dataset have that column?
+parquet_with_participantId <- lapply(all_parquet, function(parquet.df){
+  if('ParticipantIdentifier' %in% colnames(parquet.df)){
+    return(TRUE)
+  }else{
+    return(FALSE)
+  }
+})
+
+aa <- lapply(all_parquet, function(parquet.df){
```
Comment (pranavanba) [inline cohort_builder.R:70]: `aa` could have a more informative name like `parquet_with_participantId` above. Making variable names more descriptive improves code readability.

**Review by pranavanba — CHANGES_REQUESTED** [review]
See comments above

File: cohort_builder.R (hunk 1 above)
Comment (itismeghasyam) [reply cohort_builder.R:37]: This function does not exist for synapserutils (R version)

**Pull request closed without merging**